README.markdown

```diff
@@ -2721,6 +2721,8 @@ lua_need_request_body
 
 **phase:** *depends on usage*
 
+Due to the stream processing feature of HTTP2, it does not support HTTP2 connection.
```

Comment:
we need test cases to cover the new changes.

Reply:
I don't know how to start http2 client in test-nginx, Can you help to give me a example?

Reply:
set  $ENV{TEST_NGINX_USE_HTTP2}=1 in the test file.

For example:
https://github.com/openresty/lua-nginx-module/blob/master/t/163-exit-worker-hup.t#L10